Source/WebCore/css/parser/CSSParserImpl.h

```diff
@@ -91,6 +92,7 @@ class CSSParserImpl {
         CounterStyleRules,
         FontFeatureValuesRules,
         ViewTransitionRules,
+        PositionTryRules, // FIXME: is this needed??
         NoRules, // For parsing at-rules inside declaration lists (without nesting support)
```

Comment:
CounterStyle/ViewTransition/PositionTry should actually just use `NoRules` since you can't nest further rules inside them. It is not the case however for the other ones.

Reply:
I decided to do some cleanup: https://github.com/WebKit/WebKit/pull/37163